DESCRIPTION

```diff
@@ -50,6 +50,7 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
 Suggests: 
     testthat (>= 3.0.0),
+    metacore,
```

Comment:
I don't think we should make a dependency on metacore. If metacore ever gets pulled off we run the risk of xportr getting pulled off as well.

Reply:
That makes sense, I will remove it.